experimental/trusted_shuffler/backend/src/main.rs

```diff
@@ -36,12 +39,27 @@ pub struct Opt {
     listen_address: String,
 }
 
+lazy_static! {
+    // We use START_TIME to time the time elapsed between every request and response.
+    static ref START_TIME: Instant = Instant::now();
```

Comment:
Does https://doc.rust-lang.org/std/lazy/index.html work instead of lazy_static (which we generally try to avoid)?